wsapi/wsapiV2.go

```diff
@@ -722,40 +720,55 @@ func HandleV2RawData(state interfaces.IState, params interface{}) (interface{},
 		return nil, NewInvalidHashError()
 	}
 
+	dbase := state.GetDB()
 	var block interfaces.BinaryMarshallable
-	var b []byte
-
-	if block, _ = state.FetchECTransactionByHash(h); block != nil {
-		b, _ = block.MarshalBinary()
-	} else if block, _ = state.FetchFactoidTransactionByHash(h); block != nil {
-		b, _ = block.MarshalBinary()
-	} else if block, _ = state.FetchEntryByHash(h); block != nil {
```

Comment:
you replaced the query `state.FetchEntryByHash(h)` by `dbase.FetchEntry(h)`, do you know the difference? Are you confident that doesn't change the behaviour?

Reply:
I do not know the difference and I am not confident it doesn't change the behavior. I hope someone more knowledgeable can vet that.

Reply:
@WhoSoup @carryforward

Reply:
Thanks for the input. Should both state.FetchEntryByHash and dbase.FetchEntry be called, or just state.FetchEntryByHash?

Reply:
Just the state one, it calls dbase.FetchEntry already:
https://github.com/FactomProject/factomd/blob/c0f29f1452a03e36dd1115781ceab99ef2ec3cbf/state/ack.go#L611-L653